server/modules/search/elasticsearch/engine.js

```diff
@@ -2,9 +2,7 @@ const _ = require('lodash')
 const stream = require('stream')
 const Promise = require('bluebird')
 const pipeline = Promise.promisify(stream.pipeline)
-
```

Comment:
Please restore all the empty lines that are removed in this PR. Makes the code easier to read...